packages/webdriverio/src/commands/browser/deleteCookies.js

```diff
@@ -0,0 +1,13 @@
+export default async function deleteCookies(names) {
+    const namesList = typeof names !== 'undefined' && !Array.isArray(names) ? [names] : names
+
+    if (typeof namesList === 'undefined') {
+        return await this.deleteAllCookies()
```

Comment:
no need to use `await` here. You can remove this and the `async` term in the function head